ecommerce/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -23,15 +23,15 @@
                         </p>
                     {% endif %}
                     <div class="row">
-                        <div class="col-md-2 col-sm-12 product-image">
+                        <div class="col-md-2 col-sm-12 product-image hide">
```

Comment:
Does we hide this part because of the problem fetching data from Discovery?

Reply:
I tested this locally and it's still not fetching the data correctly.